feed/views.py

```diff
@@ -1,3 +1,100 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http.response   import JsonResponse
+from django.views           import View
+
+from .models                import Feed, Comment, ImageUrl
+from user.models            import User
+from product.models         import Product
+
+class FeedEntireView(View):    
+    def get(self, request):
+        MAXIMUM_COMMENT = 2
+        feed_list   = []
+        start_pos   = int(request.GET.get('offset'))
+        div         = int(request.GET.get('limit'))
+    
+        for feed in Feed.objects.all().order_by('-id'):
+
+            # 피드 정보
+            feed_basic_data = {
+                'feed_id'               : feed.id, 
+                'feed_user'             : feed.user.nickname,
+                'created_at'            : feed.created_at,
+                'description'           : feed.description,
+                'like_number'           : feed.like_number,
+                'tag_item_number'       : feed.tag_item_number,
+                'feed_main_image'       : feed.imageurl_set.values('image_url').first()
```

Comment:
```suggestion
                'feed_id'         : feed.id, 
                'feed_user'       : feed.user.nickname,
                'created_at'      : feed.created_at,
                'description'     : feed.description,
                'like_number'     : feed.like_number,
                'tag_item_number' : feed.tag_item_number,
                'feed_main_image' : feed.imageurl_set.values('image_url').first()
```
간격 너무 넓어서 줄이시면 좋을 것 같습니다!

Reply:
간격조절하도록 하겠습니다!